src/main/nav_create_simulated_image.py

```
@@ -1405,13 +1750,301 @@ def _on_body_field(self,
+                # At least one must be enabled - re-enable inner if both are disabled
+                if 'inner_data' not in ring:
+                    ring['inner_data'] = []
+                inner_data = ring['inner_data']
+                self._get_or_create_mode1(inner_data, 100.0)
+
+                # Update checkbox and controls
+                tab_idx = self._find_tab_by_properties('ring', idx)
+                if tab_idx is not None:
+                    tab_widget = self._tabs.widget(tab_idx)
+                    if tab_widget is not None:
+                        tab_widget.inner_checkbox.blockSignals(True)  # type: ignore
```

Ruff (0.14.11)

Boolean positional value in function call

(FBT003)